### ISSUE TITLE
Enemy Air Defense Enhancement

### ISSUE THREAD

**Issue by LuizBarros99** [Savage-Game-Design/Mike-Force#164]
**Is your feature request related to a problem? Please describe.**
    Not exactly a problem, but I just think it would be nice to have an option to make the enemies (VC/NVA) have a few extra air defense systems than they normally get. This is because the ZPU-4 guns tend to be their most effective at shorter ranges, and it could be interesting to have longer range alternatives.

**Describe the solution you'd like**
    The solution I would like would be to have a Mission Parameter setting that adds a probability that hostile zones will have one dynamically spawned/generated SA-2/S-75 launcher site.

   Would also be good to make sure there are some rules to the spawning of these SA-2 sites, basically have a maximum limit of how many SA-2 sites can be spawned at once. Since there would be a chance for every hostile enemy zone to have one SA-2 site if there were no rules in place, and this could become overwhelming.
   I would also suggest one last rule that would make the game prioritize spawning SA-2 sites inside of ACTIVE enemy hostile zones. Then only after that start to attempt creating the remaining SA-2 sites on "non-active" hostile enemy zones, and prioritizing zones that are located NEAR active hostile enemy zones.

   And I would also suggest something similar for the 9K32 Strela-2/2M launcher, it could be interesting to have a Mission Parameter setting that allows enemies to spawn with said launchers. On this case it could perhaps be some hostile AA soldier(s) placed inside of the hostile HQ (guarding it), or just some hostile AA soldier(s) patrolling the enemy controlled zones.

**Describe alternatives you've considered**
   The only two alternatives to adding this feature as an optional Mission Parameters setting would be either to add it without any settings (which means it can not be easily disabled), or just to not add it at all.

**Additional context**
    I am not entirely sure if these launchers can be seen on un-modified Mike Force servers, but could be interesting to have the option to make flying a little more challenging than it already is.
    
   Finally as I already said, I wish for this feature to be OPTIONAL. And no one would be forced to play with these settings, as I wish for this feature to be possible to be disabled in the Mission Parameters settings.
